index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main">
+      <div id="title" title="dr.norman borlaug">
+        <h2>Dr. Norman Borlaug</h2>
+        <p>The man who saved a billion lives</p>
+      </div>
+      <div id="img-div">
+        <img
+          id="image"
+          src="images/tribute-page-main-image.jpg"
+          alt="Image of Mr. Norman Borlaug"
+        />
+        <p id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+          on how to increase wheat yields - part of his life-long war on hunger.
+        </p>
+      </div>
+      <div id="tribute-info">
+        <h2>Here's a time line of Dr. Borlaug's life:</h2>
```

Comment:
It is important to use classes or ids for your headings, and some other elements, because with more code it will be easier to style it with classes instead of elements.